README.md

```diff
@@ -72,7 +72,7 @@ The first user of this solution is the Network and Management Laboratory of the
  
 ## Getting source repository
 
-Autonomiccs Platform project uses Git and a mirror is hosted on <a href="https://github.com/Autonomiccs/autonomiccs-platform">GitHub</a>.
+Autonomiccs Platform project uses Git and a mirror is hosted on GitHub. To access our master source code, click <a href="https://github.com/Autonomiccs/autonomiccs-platform">here</a>. The latest stable version source code is with the tag <a href="https://github.com/Autonomiccs/autonomiccs-platform/tree/1.0.0">1.0.0</a>.

```

Comment:
what about changing the name of the link from 1.0.0 to only "stable", then we would need only to update the link for the tag when a new release is out.